contrib/redwood/plugins.yml

```diff
@@ -76,6 +128,19 @@
   description: |
     Sentry integration for Open edX.
 
+- name: webhooks
+  src: tutor-contrib-webhooks>=18.0.0,<19.0.0
+  url: https://github.com/aulasneo/tutor-contrib-webhooks
+  author: Andrés González <andres@aulasneo.com>
+  maintainer: Andrés González <andres@aulasneo.com>
+  description: |
+    Connect your Open edX instance to external services with this handful plugin.
+    by adding webhooks and webfilters.
+    Both webhooks and webfilters are triggered when certain events happen in your platform,
+    e.g. when a student logs in, when a certificate is issues, and many more. Then they will
```

Comment:
isssues -> issued

Reply:
Thanks Régis for taking the time to make these corrections!! I really appreciate it.